enos/modules/vault_cluster/main.tf

```diff
@@ -15,6 +15,10 @@ terraform {
 data "enos_environment" "localhost" {}
 
 locals {
+  distro_version_sles = {
```

Comment:
I don't see where we use this. Also, we should be getting this from variables, right?

Reply:
yep this is leftover from earlier attempts. Removed.